src/main/java/com/cloudant/fdblucene/FDBTermsEnum.java

```diff
@@ -60,7 +63,16 @@ public FDBTermsEnum(final Transaction txn, final Subspace index, final String fi
 
     @Override
     public BytesRef next() throws IOException {
-        throw new UnsupportedOperationException("next not supported.");
+        final KeySelector begin = KeySelector.firstGreaterThan(FDBAccess.docFreqKey(index, fieldName, term));
```

Comment:
this range here solves querying with prefixes?

Reply:
that range finds the next term after then one we're on, or it hits the end of the range. 

Implementing next() allows all PrefixQuery's to work. Some of them work with just seekExact (i.e, our unit test only required that). I discovered this gap by running various searches in the grpcbox shell.